core/src/main/java/com/novoda/merlin/Endpoint.java

```diff
@@ -5,7 +5,7 @@
 
 public class Endpoint {
 
-    private static final Endpoint DEFAULT_ENDPOINT = Endpoint.from("http://connectivitycheck.android.com/generate_204");
+    private static final Endpoint DEFAULT_ENDPOINT = Endpoint.from("https://connectivitycheck.android.com/generate_204");
```

Comment:
👍 As @ouchadam said in #172 I think this is the best option to support Android 9.0